### PR TITLE
KOGITO-2547: Include Webview into DefaultVsCodeRouter

### PR DESCRIPTION
**Pull request by caponetto**

### CODE REVIEW

**Review by paulovmr — APPROVED** [review]

File: packages/vscode-extension/src/KogitoEditorStore.ts [hunk 1]
```diff
@@ -81,8 +81,4 @@ export class KogitoEditorStore {
 
     return found;
   }
-
-  public getActive() {
-    return this.activeEditor;
-  }
```
Comment (tiagobento) [inline packages/vscode-extension/src/KogitoEditorStore.ts:87]: 😍

**Review by tiagobento — APPROVED** [review]
Nice! Thanks for the PR @caponetto. Looks great. Let's wait for the QE approve.

File: packages/vscode-extension/src/DefaultVsCodeRouter.ts [hunk 2]
```diff
@@ -15,27 +15,15 @@
...
-      "vscodePathResolver"
-    );
-
-    if (vscodePathResolver) {
-      return vscodePathResolver(vscode.Uri.file(this.context.asAbsolutePath(uri))).toString();
-    }
-    // TODO Figure out a better way to do this fallback for when the Webview#asWebviewUri is not available yet
-    return "vscode-resource://" + vscode.Uri.file(this.context.asAbsolutePath(uri)).toString().replace(":", "");
+  public getRelativePathTo(uri: string): string {
+    return this.webview.asWebviewUri(Uri.file(this.context.asAbsolutePath(uri))).toString();
```
Comment (jomarko) [inline packages/vscode-extension/src/DefaultVsCodeRouter.ts:26]: I think we should add a unit test for this change. Or could you point me to test of this if such exists?

**Review by jomarko — COMMENTED** [review]
One comment to code.

Also please notice tester field in the jira. We should ask @domhanak if he is ok with merge or change tester in the jira.

File: packages/vscode-extension/src/DefaultVsCodeRouter.ts (hunk 2 above)
Comment (caponetto) [reply packages/vscode-extension/src/DefaultVsCodeRouter.ts:26]: There isn't a test for this. All involved objects are not under our control (`Webview`, `Uri`, `ExtensionContext`), so a test for this change would be just a bunch of mocks doing everything we say. Do you think it's relevant?

Comment (jomarko) [reply packages/vscode-extension/src/DefaultVsCodeRouter.ts:26]: @caponetto fine for me, as said above, @domhanak works primarily on VS Code and working on come integration tests so please do not merge without his approval.

**Review by jomarko — APPROVED** [review]

**Comment by caponetto** [comment]
@jomarko When a JIRA is created, @domhanak is the default assigned tester.
I guess it is a matter of changing it when another QE is asked to review on Github. 
Wdyt?
@tiagobento thoughts?

**Comment by jomarko** [comment]
@caponetto I saw in some document from our meetings that tester in jira == tester on PR (usually, for sure there are exceptions). 
I can take this one, however still think:
- Dominik knows more about VS code channel
- We should have jira and github PR in corresponding state

**Review by jomarko — APPROVED** [review]
Didn't found a regression in VSCode channel by manual testing.

**Pull request merged**